python_bindings/tests/end-to-python-end-test/two_player_game_test.py

```diff
@@ -0,0 +1,572 @@
+           ..|...\ /.!.|....|..     
+0          ..4----0----3....B--     
+           ..|.................     
+           ..|.................     
+           ..|..     -...--...--...-
+           .[ ].      \./  \./  \./
+-1         .[5].       6    7    A @
+           .[!].      /.\  /.\  /.\ 
+           .....     -...--   --...-
+"""
+
+
+def check_eleventh_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=2, y=0),
+        tile=game.current_tile,
+        meepleType=MeepleType.NoneMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.NoneType,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [8, 12])
+
+    return game_id, game
+
+
+"""
+// straight road
+            -1    0    1    2    3
+            
+            
+           ..|.!............|..     
+           ..|..............|..     
+1          ..9----1----2....8..     
+           ..|.../ \...|....|..     
+           ..|..|   |..|....@..     
+           ..|..|   |..|....|.......
+           ..|...\ /.!.|....|.......
+0          ..4----0----3....B----C--
+           ..|......................
+           ..|......................
+           ..|..     -...--...--...-
+           .[ ].      \|/  \./  \./
+-1         .[5].       6    7    A @
+           .[!].      /.\  /.\  /.\ 
+           .....     -...--   --...-
+"""
+
+
+def check_twelfth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=3, y=0),
+        tile=tiletemplates.straight_roads(),
+        meepleType=MeepleType.NoneMeeple,
+        side=Side.NoSide,
+        featureType=FeatureType.NoneType,
+    )
+
+    game_id, game = make_turn(
+        engine, game, game_id, turn_params, True, {1: 8 + 12, 2: 12 + 4}
+    )
+    check_points(game, [8, 12])
+
+    return game_id, game
+
+
+"""
+player1 scores 11 points in total:
+    - 2*3points for farmer on 9 tile
+    - 3 points for farmer on 3 tile
+    - 3 points for monastery on 5 tile  
+
+player2 scores 4 points in total:
+    - 3 points for road on 8 tile
+    - 1 point for city on A tile    
+
+            -1    0    1    2    3
+
+
+           ..|.!............|..     
+           ..|..............|..     
+1          ..9----1----2....8..     
+           ..|.../ \...|....|..     
+           ..|..|   |..|....@..     
+           ..|..|   |..|....|.......
+           ..|...\ /.!.|....|.......
+0          ..4----0----3....B----C--
+           ..|......................
+           ..|......................
+           ..|..     -...--...--...-
+           .[ ].      \|/  \./  \./
+-1         .[5].       6    7    A @
+           .[!].      /.\  /.\  /.\ 
+           .....     -...--   --...-
+"""
+
+
+def finalize(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    check_final_points([8 + 12, 12 + 4])
+    return
```

Comment:
Similar problems as in the previous finalize()

Reply:
removed